docs/conf.py

```diff
@@ -222,3 +222,19 @@
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
+
+# -- Ignore warnings ---------------------------------------------------------
+
+# Ignore reference targets not found
+nitpick_ignore = [
+  ('py:func', 'BuildStockBatchBase.validate_precomputed_sample'),
+  ('py:func', 'BuildStockBatchBase.validate_xor_nor_schema_keys'),
+  ('py:func', 'EagleBatch.run_building'),
+  ('py:class', 'sampler.CommercialSobolSingularitySampler'),
+  ('py:class', 'sampler.CommercialSobolDockerSampler'),
+  ('py:class', 'workflow_generator.CommercialDefaultWorkflowGenerator'),
+  ('py:class', 'sampler.PrecomputedSampler'),
+  ('py:class', 'sampler.BuildStockSampler'),
+  ('py:class', 'BuildStockBatchBase'),
+  ('py:func', 'BuildStockBatchBase.run_sampling')
+]
```

Comment:
Any idea why these aren't being found?

Reply:
I think these are all in old changelog files, and do not exist anymore.

Reply:
Here is the failed ci that lists them: https://github.com/NREL/buildstockbatch/runs/7046343242?check_suite_focus=true